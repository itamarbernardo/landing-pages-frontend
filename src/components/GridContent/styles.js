import styled, { css } from 'styled-components';
import { Title } from '../Heading/styles';

export const Container = styled.div`
  ${({ theme, background }) => css`
    text-align: center;
    max-width: 58rem;
    margin: 0 auto;

    ${Title} {
      color: ${background ? theme.colors.white : theme.colors.primaryColor};
    }
  `}
`;

export const Html = styled.div`
  ${({ theme }) => css`
    margin: ${theme.spacings.xhuge} 0;
  `}
`;
