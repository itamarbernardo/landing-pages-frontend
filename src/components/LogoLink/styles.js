import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    text-decoration: none;
    //color: inherit;
    color: 'black';
    > img {
      height: 3rem;
    }
  `}
`;
