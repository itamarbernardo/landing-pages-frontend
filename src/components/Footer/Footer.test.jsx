import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { Footer } from '.';

describe('<Footer />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Footer footerHtml={'<h1>Olá tudo bem</h1>'} />);
    //expect(screen.getByRole('heading', { name: 'Olá tudo bem' })).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
