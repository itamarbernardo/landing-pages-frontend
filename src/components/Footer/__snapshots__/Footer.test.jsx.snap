// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
.c4 {
  font-size: 1.6rem;
}

.c2 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c0 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c0 a {
  color: inherit;
  text-decoration: none;
}

.c0 .c3 {
  font-size: 1.6rem;
}

.c0 .c1 {
  padding-top: 0;
  padding-bottom: 0;
}

<div>
  <footer
    class="c0"
  >
    <div
      class="c1 c2"
    >
      <h1>
        <p
          class="c3 c4"
        >
          <h1>
            Ol√° tudo bem
          </h1>
        </p>
      </h1>
    </div>
  </footer>
</div>
`;
